cost-analyzer/templates/aggregator-service.yaml

```diff
@@ -16,5 +16,8 @@ spec:
     - name: tcp-api
       port: 9004
       targetPort: 9004
+      {{- with .Values.kubecostModel.extraPorts }}
```

Comment:
`.Values.kubecostAggregator.extraPorts` I think makes more sense. Same deal for other aggregator files.